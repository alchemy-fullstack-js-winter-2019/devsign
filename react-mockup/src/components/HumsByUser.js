import React from 'react';

function HumsByUser() {
  return (
    <section>
      <form>
        <input type="text" name="hum" />
        <button>Post new hum</button>
      </form>
      <div>
        <p>Hum hum hum. bother. bother. bother.</p>
      </div>
      <hr />
      <div>
        <p>Hum hum hum. bother. bother. bother.</p>
      </div>
      <hr />
      <div>
        <p>Hum hum hum. bother. bother. bother.</p>
      </div>
    </section>
  );
}

export default HumsByUser;
