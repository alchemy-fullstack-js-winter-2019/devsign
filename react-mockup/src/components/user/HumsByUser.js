import React from 'react';
import styled from 'styled-components';

const Section = styled.section`
  font-family: sans-serif;
  border: 1px solid gray;
  padding-top: 10px;
  h3 {
    font-family: 'Montserrat', sans-serif; 
    text-align: center;
  }
  div {
    border-bottom: 1px solid gray;
    padding: 15px;
    margin-bottom: 5px;
  }
`;

function HumsByUser() {
  return (
    <Section>
      <h3>hums by you!</h3>
      <div>
        <p>Hum hum hum. bother. bother. bother.</p>
      </div>
      <div>
        <p>Hum hum hum. bother. bother. bother.</p>
      </div>
      <div>
        <p>Hum hum hum. bother. bother. bother.</p>
      </div>
    </Section>
  );
}

export default HumsByUser;
