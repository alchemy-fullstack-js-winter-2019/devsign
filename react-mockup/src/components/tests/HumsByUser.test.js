import React from 'react';
import renderer from 'react-test-renderer';
import HumsByUser from '../HumsByUser';
import { MemoryRouter } from 'react-router';

describe('HumsByUser snapshot', () => {
  it('can match a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <HumsByUser />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
