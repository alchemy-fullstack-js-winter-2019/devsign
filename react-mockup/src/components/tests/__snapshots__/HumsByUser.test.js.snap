// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HumsByUser snapshot can match a snapshot 1`] = `
<section
  className="sc-bxivhb hggFOb"
>
  <h3>
    hums by user!
  </h3>
  <div>
    <p>
      Hum hum hum. bother. bother. bother.
    </p>
  </div>
  <div>
    <p>
      Hum hum hum. bother. bother. bother.
    </p>
  </div>
  <div>
    <p>
      Hum hum hum. bother. bother. bother.
    </p>
  </div>
  <div>
    <p>
      Hum hum hum. bother. bother. bother.
    </p>
  </div>
  <div>
    <p>
      Hum hum hum. bother. bother. bother.
    </p>
  </div>
  <div>
    <p>
      Hum hum hum. bother. bother. bother.
    </p>
  </div>
</section>
`;
